android/src/main/java/com/cuhacking/atlas/MainActivity.kt

```diff
@@ -13,6 +20,70 @@ class MainActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        binding.exampleText.text = getMessage()
+        binding.mapView.onCreate(savedInstanceState)
+
+        binding.mapView.getMapAsync { map ->
+            map.setStyle(Style.DARK) { style ->
+                // I took some artistic liberties while figuring out how to do this
+
+                // FeatureCollection from MultiPolygon GeoJson
+                val featureCollection = FeatureCollection.fromJson(giveJson())
+                val geoJsonSource = GeoJsonSource("geojson-source", featureCollection)
+                style.addSource(geoJsonSource)
+
+                val fillLayer = FillLayer("fill-layer", "geojson-source")
+                fillLayer.setProperties(PropertyFactory.fillColor(Color.DKGRAY))
+                style.addLayer(fillLayer)
+
+                //! https://stackoverflow.com/a/40286827
+                //  not android sdk but originally found below
+                //! https://github.com/mapbox/mapbox-gl-js/issues/3018
+                val outlineLayer = LineLayer("outline-layer", "geojson-source")
+                outlineLayer.setProperties(
+                    PropertyFactory.lineColor(
+                        ContextCompat.getColor(
+                            applicationContext,
```

Comment:
This is sus, but I'll let it slide since this code is only temporary.

```suggestion
                            this,
```